 2019-09-23 02:10:20,130 [P2952/D2/T147] INFO  umbraco.BusinessLogic.Log - Log scrubbed.  Removed all items older than 2019-09-22 02:10:20
 2019-09-23 06:10:20,318 [P2952/D2/T167] INFO  umbraco.BusinessLogic.Log - Log scrubbed.  Removed all items older than 2019-09-22 06:10:20
 2019-09-23 07:06:14,442 [P2952/D2/T73] ERROR Umbraco.Web.Scheduling.KeepAlive - Failed (at "http://83.41.86.152:8008/umbraco").
System.Net.Http.HttpRequestException: Error al enviar la solicitud. ---> System.Net.WebException: No es posible conectar con el servidor remoto ---> System.Net.Sockets.SocketException: Se produjo un error durante el intento de conexión ya que la parte conectada no respondió adecuadamente tras un periodo de tiempo, o bien se produjo un error en la conexión establecida ya que el host conectado no ha podido responder 83.41.86.152:8008
   en System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   en System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   en System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- Fin del seguimiento de la pila de la excepción interna ---
   en System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)
   en System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)
   --- Fin del seguimiento de la pila de la excepción interna ---
   en System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   en System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   en Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()
 2019-09-23 07:11:35,552 [P2952/D2/T73] ERROR Umbraco.Web.Scheduling.KeepAlive - Failed (at "http://83.41.86.152:8008/umbraco").
System.Net.Http.HttpRequestException: Error al enviar la solicitud. ---> System.Net.WebException: No es posible conectar con el servidor remoto ---> System.Net.Sockets.SocketException: Se produjo un error durante el intento de conexión ya que la parte conectada no respondió adecuadamente tras un periodo de tiempo, o bien se produjo un error en la conexión establecida ya que el host conectado no ha podido responder 83.41.86.152:8008
   en System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   en System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   en System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- Fin del seguimiento de la pila de la excepción interna ---
   en System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)
   en System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)
   --- Fin del seguimiento de la pila de la excepción interna ---
   en System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   en System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   en Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()
 2019-09-23 07:16:56,614 [P2952/D2/T73] ERROR Umbraco.Web.Scheduling.KeepAlive - Failed (at "http://83.41.86.152:8008/umbraco").
System.Net.Http.HttpRequestException: Error al enviar la solicitud. ---> System.Net.WebException: No es posible conectar con el servidor remoto ---> System.Net.Sockets.SocketException: Se produjo un error durante el intento de conexión ya que la parte conectada no respondió adecuadamente tras un periodo de tiempo, o bien se produjo un error en la conexión establecida ya que el host conectado no ha podido responder 83.41.86.152:8008
   en System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   en System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   en System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- Fin del seguimiento de la pila de la excepción interna ---
   en System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)
   en System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)
   --- Fin del seguimiento de la pila de la excepción interna ---
   en System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   en System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   en Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()
 2019-09-23 07:21:05,520 [P2952/D2/T80] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [HealthCheckNotifier] Terminating
 2019-09-23 07:21:05,520 [P2952/D2/T80] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [HealthCheckNotifier] Waiting for tasks to complete
 2019-09-23 07:21:05,520 [P2952/D2/T80] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [HealthCheckNotifier] Tasks completed, terminated
 2019-09-23 07:21:05,520 [P2952/D2/T80] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [ScheduledTasks] Terminating
 2019-09-23 07:21:05,520 [P2952/D2/T80] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [ScheduledTasks] Waiting for tasks to complete
 2019-09-23 07:21:05,520 [P2952/D2/T80] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [InstructionProcessing] Terminating
 2019-09-23 07:21:05,538 [P2952/D2/T80] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [InstructionProcessing] Waiting for tasks to complete
 2019-09-23 07:21:05,538 [P2952/D2/T80] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [Umbraco.Web.Scheduling.IBackgroundTask] Terminating
 2019-09-23 07:21:05,538 [P2952/D2/T80] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [Umbraco.Web.Scheduling.IBackgroundTask] Waiting for tasks to complete
 2019-09-23 07:21:05,538 [P2952/D2/T80] INFO  Umbraco.Core.MainDom - Stopping...
 2019-09-23 07:21:05,552 [P2952/D2/T166] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [ScheduledTasks] Tasks completed, terminated
 2019-09-23 07:21:05,552 [P2952/D2/T162] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [Umbraco.Web.Scheduling.IBackgroundTask] Tasks completed, terminated
 2019-09-23 07:21:05,552 [P2952/D2/T138] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [InstructionProcessing] Tasks completed, terminated
 2019-09-23 07:21:05,552 [P2952/D2/T80] INFO  Umbraco.Web.ExamineStartup - Examine shutting down
 2019-09-23 07:21:05,552 [P2952/D2/T80] INFO  Umbraco.Web.ExamineStartup - Complete (took 2ms)
 2019-09-23 07:21:05,552 [P2952/D2/T80] INFO  Umbraco.Core.MainDom - Released MainDom.
 2019-09-23 07:21:05,552 [P2952/D2/T80] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [ScheduledPublishing] Terminating
 2019-09-23 07:21:05,552 [P2952/D2/T80] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [ScheduledPublishing] Waiting for tasks to complete
 2019-09-23 07:21:05,552 [P2952/D2/T75] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [ScheduledPublishing] Tasks completed, terminated
 2019-09-23 07:21:05,584 [P2952/D2/T80] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [KeepAlive] Terminating
 2019-09-23 07:21:05,584 [P2952/D2/T80] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [KeepAlive] Waiting for tasks to complete
 2019-09-23 07:21:05,584 [P2952/D2/T80] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [LogScrubber] Terminating
 2019-09-23 07:21:05,584 [P2952/D2/T137] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [KeepAlive] Tasks completed, terminated
 2019-09-23 07:21:05,584 [P2952/D2/T80] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [LogScrubber] Waiting for tasks to complete
 2019-09-23 07:21:05,584 [P2952/D2/T168] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [LogScrubber] Tasks completed, terminated
 2019-09-23 07:21:05,708 [P2952/D2/T57] INFO  Umbraco.Core.UmbracoApplicationBase - Application shutdown. Details: HostingEnvironment

_shutDownMessage=HostingEnvironment initiated shutdown
HostingEnvironment originó el cierre

_shutDownStack=   en System.Environment.GetStackTrace(Exception e, Boolean needFileInfo)
   en System.Environment.get_StackTrace()
   en System.Web.Hosting.HostingEnvironment.InitiateShutdownInternal()
   en System.Web.Hosting.HostingEnvironment.InitiateShutdownWithoutDemand()
   en System.Web.Hosting.PipelineRuntime.StopProcessing()
 2019-09-23 16:04:23,067 [P3144/D2/T1] INFO  Umbraco.Core.CoreBootManager - Umbraco 7.15.3 application starting on SYNOVMW10-UMBRA
 2019-09-23 16:04:23,146 [P3144/D2/T1] INFO  Umbraco.Core.PluginManager - Determining hash of code files on disk
 2019-09-23 16:04:23,177 [P3144/D2/T1] INFO  Umbraco.Core.PluginManager - Hash determined (took 37ms)
 2019-09-23 16:04:23,364 [P3144/D2/T1] INFO  Umbraco.Core.MainDom - Acquiring MainDom...
 2019-09-23 16:04:23,364 [P3144/D2/T1] INFO  Umbraco.Core.MainDom - Acquired MainDom.
 2019-09-23 16:04:23,381 [P3144/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-09-23 16:04:23,630 [P3144/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 257ms)
 2019-09-23 16:04:23,630 [P3144/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IApplicationStartupHandler
 2019-09-23 16:04:23,647 [P3144/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IApplicationStartupHandler (took 6ms)
 2019-09-23 16:04:23,756 [P3144/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-09-23 16:04:23,756 [P3144/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 2ms)
 2019-09-23 16:04:23,756 [P3144/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.PropertyEditors.IPropertyEditorValueConverter
 2019-09-23 16:04:23,756 [P3144/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.PropertyEditors.IPropertyEditorValueConverter (took 1ms)
 2019-09-23 16:04:23,756 [P3144/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-09-23 16:04:23,756 [P3144/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 1ms)
 2019-09-23 16:04:23,770 [P3144/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.PropertyEditors.IPropertyValueConverter
 2019-09-23 16:04:23,770 [P3144/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.PropertyEditors.IPropertyValueConverter (took 6ms)
 2019-09-23 16:04:23,802 [P3144/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-09-23 16:04:23,802 [P3144/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 3ms)
 2019-09-23 16:04:23,802 [P3144/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Web.Mvc.SurfaceController
 2019-09-23 16:04:23,802 [P3144/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Web.Mvc.SurfaceController (took 2ms)
 2019-09-23 16:04:23,802 [P3144/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-09-23 16:04:23,802 [P3144/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 1ms)
 2019-09-23 16:04:23,802 [P3144/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Web.WebApi.UmbracoApiController
 2019-09-23 16:04:23,802 [P3144/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Web.WebApi.UmbracoApiController (took 3ms)
 2019-09-23 16:04:32,849 [P3144/D2/T1] INFO  Umbraco.Core.DatabaseContext - CanConnect = True
 2019-09-23 16:04:33,474 [P3144/D2/T1] INFO  Umbraco.Web.Cache.CacheRefresherEventHandler - Initializing Umbraco internal event handlers for cache refreshing
 2019-09-23 16:04:33,864 [P3144/D2/T1] INFO  Umbraco.Web.Search.ExamineEvents - Initializing Examine and binding to business logic events
 2019-09-23 16:04:33,864 [P3144/D2/T1] INFO  Umbraco.Web.Search.ExamineEvents - Adding examine event handlers for index providers: 3
 2019-09-23 16:04:34,054 [P3144/D2/T1] INFO  Umbraco.Core.CoreBootManager - Umbraco application startup complete (took 11265ms)
 2019-09-23 16:04:34,817 [P3144/D2/T11] INFO  Umbraco.Core.Sync.ApplicationUrlHelper - New ApplicationUrl detected: http://83.40.126.69:8008/umbraco
 2019-09-23 16:04:34,817 [P3144/D2/T11] INFO  Umbraco.Core.Sync.ApplicationUrlHelper - ApplicationUrl: http://83.40.126.69:8008/umbraco (UmbracoModule request)
 2019-09-23 16:04:36,536 [P3144/D2/T11] INFO  umbraco.content - Load Xml from file...
 2019-09-23 16:04:36,536 [P3144/D2/T11] INFO  umbraco.content - Loaded Xml from file.
 2019-09-23 16:04:39,588 [P3144/D2/T11] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-09-23 16:04:39,588 [P3144/D2/T11] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 1ms)
 2019-09-23 16:04:39,588 [P3144/D2/T11] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.PropertyEditors.IParameterEditor
 2019-09-23 16:04:39,599 [P3144/D2/T11] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.PropertyEditors.IParameterEditor (took 6ms)
 2019-09-23 16:04:41,291 [P3144/D2/T11] WARN  Umbraco.Web.Mvc.RenderRouteHandler - The current Document Type home matches a locally declared controller of type UmbracoTesting.Controllers.HomeController. Custom Controllers for Umbraco routing must implement 'Umbraco.Web.Mvc.IRenderController' and inherit from 'System.Web.Mvc.ControllerBase'.
 2019-09-23 16:05:36,692 [P3144/D2/T8] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-09-23 16:05:36,708 [P3144/D2/T8] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 1ms)
 2019-09-23 16:05:36,708 [P3144/D2/T8] INFO  Umbraco.Core.PluginManager - Resolving [Umbraco.Core.Persistence.Mappers.MapperForAttribute]Umbraco.Core.Persistence.Mappers.BaseMapper
 2019-09-23 16:05:36,708 [P3144/D2/T8] INFO  Umbraco.Core.PluginManager - Resolved [Umbraco.Core.Persistence.Mappers.MapperForAttribute]Umbraco.Core.Persistence.Mappers.BaseMapper (took 2ms)
 2019-09-23 16:05:36,729 [P3144/D2/T11] INFO  umbraco.BusinessLogic.Log - Log scrubbed.  Removed all items older than 2019-09-22 16:05:36
 2019-09-23 20:05:37,385 [P3144/D2/T86] INFO  umbraco.BusinessLogic.Log - Log scrubbed.  Removed all items older than 2019-09-22 20:05:36
 2019-09-23 21:08:36,114 [P3144/D2/T87] WARN  Umbraco.Web.Mvc.RenderRouteHandler - The current Document Type home matches a locally declared controller of type UmbracoTesting.Controllers.HomeController. Custom Controllers for Umbraco routing must implement 'Umbraco.Web.Mvc.IRenderController' and inherit from 'System.Web.Mvc.ControllerBase'.
